deeplearning4j-core/src/test/java/org/deeplearning4j/gradientcheck/CNN1DGradientCheckTest.java

```diff
@@ -51,7 +51,7 @@ public void testCnn1DWithZeroPadding1D() {
         int zeroPadding = 2;
         int paddedLength = length + 2 * zeroPadding;
 
-        Activation[] activations = {Activation.SIGMOID, Activation.TANH};
```

Comment:
I want this one to keep failing so it doesn't get forgotten/missed :)

Reply:
Since we have an open ticket with you assigned already (https://github.com/deeplearning4j/deeplearning4j/issues/3975) and I'd rather keep the build green, can we agree to leave it like this?